using-mobile-extensions/adobe-target-vec.md

```diff
@@ -0,0 +1,144 @@
+
+#### Objective-C
+
+Import the Target and Identity library.
+
+```objectivec
+   #import "ACPCore.h"
+   #import "ACPTargetVEC.h"
+   #import "ACPTarget.h"
+   #import "ACPIdentity.h"
+   #import "ACPTargetRequestObject.h"
+   #import "ACPTargetPrefetchObject.h"
+```
+
+#### Swift
+
+```swift
+   #import ACPCore
+   #import ACPTarget
+   #import ACPTargetVEC
+   #import ACPIdentity
+```
+
+### Register Target VEC with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+#### Java
+
+After calling the `setApplication()` method in the `onCreate()` method, register Target VEC with Mobile Core.
+
+Here is code sample that calls these set up methods:
+
+```java
+public class TargetApp extends Application {
+
+ @Override
+ public void onCreate() {
+     super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         //Other Extensions that you need
+         TargetVEC.registerExtension();
+         Target.registerExtension();
+     } catch (Exception e) {
+         //Log the exception
+     }
+ }
+}
+```
+{% endtab %}
+
+{% tab title="iOS" %}
+#### Objective-C
+
+1. In your app's `didFinishLaunchingWithOptions` function register the Target VEC extension
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  //Other Extensions that you need
+  [ACPTargetVEC registerExtension];
+  [ACPTarget registerExtension];
+
+  // Override point for customization after application launch.
+  return YES;
+}
+```
+
+#### Swift
+1. In your app's `didFinishLaunchingWithOptions` function register the Target VEC extension
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  //Other Extensions that you need
+  ACPTargetVEC.registerExtension()
+  ACPTarget.registerExtension()
+  
+  return true
+}
+```
+{% endtab %}
+{% endtabs %}
+
+## Implementation Methods for Target VEC   <a id="implementation-methods-for-target-vec"></a>
+The Target VEC extension retrieves the relevant Target experiences for your app through a single network request. Offers are retrieved via this network call and applied automatically on the targeted screens. It should be noted that no network requests are made to retrieve VEC experiences as the user navigates through multiple screens of the app.
+
+The default behavior of the extension is to make a synchronous network request (blocking call) at the time of Application launch. You can use Launch to control the behavior of this network request to meet your application behavior.
+
+### Auto-Fetch Target Activities
+This is the default behavior where a network request is initiated automatically by the Target VEC extension. You can use one of the following options to make this request a blocking call or an asynchronous request.
+
+#### Fetch in a synchronous call (background is OFF)
```

Comment:
as "Fetch in a synchronous call" is an option of "Auto-Fetch Target Activites", I'd recommend putting both of the "Fetch" options in a bulleted list like this:

### Auto-Fetch Target Activities

 * Fetch in a synchronous call (background is OFF)
* Fetch in an asynchronous call (background is ON)